templates/frontend/pages/submissions.tpl

```diff
@@ -83,6 +58,21 @@
 	</div>
 	{/if}
 
+	{foreach from=$sections item="section"}
+		{if $section->getLocalizedPolicy()}
+			<div class="section_policy">
+				<h2>{$section->getLocalizedTitle()|escape}</h2>
+				{$section->getLocalizedPolicy()}
+				{if $isUserLoggedIn && !$section->getEditorRestricted()}
```

Comment:
The condition `!$section->getEditorRestricted()` prevents the display of the "make a submission" link when the user is an editor and is logged in. I think its better to remove this condition as it would confuse "editors" when visiting the about submissions page due to a possibly displayed section but a missing submission link. Also, editor restriction is taken care of in `sectionDao->getByContextId()` already. What do you say?

Reply:
Ah, good catch! You're right, `sectionDao->getByContextId()` covers it here. That means that we can get rid of lines 95-101 and 110 of `AboutContextHandler` as well.

In the template file, you can then replace `$hasSubmittableSections` with `count($sections)`.

:+1: